package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "surrealdb.js",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
Could you remove this new version (and in the package-lock.json), and I'll bump it separately (with a Git tag too), and update dependencies?